doc/libpmemobj/pmemobj_ctl_get.3.md

```diff
@@ -320,6 +320,19 @@ The required class identifier will be stored in the `class_id` field of the
 This function returns 0 if the allocation class has been successfully created,
 -1 otherwise.
 
+heap.size.granularity | rw- | - | size_t | size_t | - | long long
+
+Reads or modifies the granularity with which the heap grows when OOM.
+Valid only if the poolset has been defined with directories.
+
+This function returns 0 is the value is larger than *PMEMOBJ_MIN_POOL*,
+-1 otherwise.
+
+heap.size.extend | --x | - | - | - | size_t | -
+
+Extends the heap by the given size. Must be larger than *PMEMOBJ_MIN_POOL*.
```

Comment:
Why PMEMOBJ_MIN_POOL? Can we get separate constant for that? Why not PAGE_SIZE?

Reply:
It should be larger than a single chunk, otherwise you won't see the difference.
I don't see a reason why we should have yet another variable - I don't think anyone will use this in megabyte ranges but rather in gigabytes.

Reply:
If single chunk size it the minimum size than let's state that. PMEMOBJ_MIN_POOL is an arbitrary value. If we'll ever need to change the minimum pool size then it will also affect minimum extend size for no good reason.

I think it would be fine to say that "PAGE_SIZE is the minimum, but multiples of 256k are the most optimal" (without introducing named constant for 256k).